### PR TITLE
Give more verbosity for CI

### PR DESCRIPTION
**Pull request by pmbrull**
### Describe your changes :

I worked on the ..... because ...

#
### Type of change :

- [x] Bug fix
- [x] Improvement
- [x] New feature
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [] Documentation

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: docker/run_local_docker.sh [hunk 1]
```diff
@@ -19,12 +19,12 @@ echo "Starting Local Docker Containers"
 
 docker compose down && docker compose up --build -d
 
-until curl -s -f -o /dev/null "http://localhost:9200/_cat/indices/team_search_index"; do
-  printf '.'
+until curl -s -f "http://localhost:9200/_cat/indices/team_search_index"; do
+  printf 'Checking if ES is up...\n'
```
Comment (vivekratnavel) [inline docker/run_local_docker.sh:23]: Nit:
```suggestion
  printf 'Checking if Elastic Search instance is up...\n'
```

File: docker/run_local_docker.sh [hunk 2]
```diff
@@ -33,9 +33,16 @@ curl --location --request PATCH 'localhost:8080/api/v1/dags/sample_data' \
...
-  printf '.'
-  sleep 2
+
+cd ../
+printf 'Validate sample data DAG...'
+sleep 5
+python validate_compose.py
+
+until curl -s -f "http://localhost:8585/api/v1/tables/name/sample_data.ecommerce_db.shopify.fact_sale"; do
+  printf 'Waiting on Sample Data Ingestion...\n'
```
Comment (vivekratnavel) [inline docker/run_local_docker.sh:43]:
```suggestion
  printf 'Waiting on Sample Data Ingestion to complete...\n'
```

File: docker/validate_compose.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    ).text
+    pprint(logs)
+
+
+def main():
+
+    state = None
+    while state != "success":
+
+        print("Waiting for sample data to be a success. We'll show some logs along the way.")
```
Comment (vivekratnavel) [inline docker/validate_compose.py:55]:
```suggestion
        print("Waiting for sample data ingestion to be a success. We'll show some logs along the way.")
```

**Review by vivekratnavel — APPROVED** [review]
Few minor comments

File: docker/run_local_docker.sh (hunk 1 above)
Comment (pmbrull) [reply docker/run_local_docker.sh:23]: all makes sense, thanks. Will update that

**Pull request merged**